Bio/SeqIO/FastaIO.py

```diff
@@ -202,7 +205,7 @@ def __init__(
         """
         if alphabet is not None:
             raise ValueError("The alphabet argument is no longer supported")
-        super().__init__(source, mode="t", fmt="Fasta")
+        super().__init__(source, mode="t", fmt="FASTA")
```

Comment:
Why now in capitals? All the SeqIO format names are lower case.

Reply:
In Biopython 1.84, `FastaIterator` is using `fmt="Fasta"` while `FastaTwoLineIterator` is using `fmt="FASTA"`. I don't mind either way, but it's good to be consistent.

Reply:
They should probably both be lower case then, I don't immediately recall where that is used though.

Reply:
I'll revert this and let someone else figure this out